perf/benchmark_radix_tree.cpp

```diff
@@ -104,6 +109,8 @@ int main ()
     }
 
     // Create a benchmark.
+    std::printf ("keys = %ld, queries = %ld, key size = %ld\n", nkeys, nqueries,
+                 key_length);
```

Comment:
This fails to compile with some combinations of gcc/libc:

```
perf/benchmark_radix_tree.cpp:113:28: error: format '%ld' expects argument of type 'long int', but argument 2 has type 'std::size_t {aka unsigned int}' [-Werror=format=]
                  key_length);
```

Reply:
Used wrong format specifier, using `%lu` now

Reply:
still not happy:

```
perf/benchmark_radix_tree.cpp:113:28: error: format '%lu' expects argument of type 'long unsigned int', but argument 2 has type 'std::size_t {aka unsigned int}' [-Werror=format=]
```

You probably want to use llu and cast the variables to (unsigned long long) for portability

Reply:
Looks like `%zu` is the correct specifier but it might not be supported in some cases. I'll just change the variable type to unsigned long since the build still fails.

Reply:
Using a cast sounds like a better idea